src/ops/ops.go

```diff
@@ -181,6 +181,15 @@ func (o *ops) SetBootOrder(device string) error {
 		return nil
 	}
 
+	o.log.Infof("SetBootOrder, runtime.GOARCH: %s, device: %s", runtime.GOARCH, device)
+	if runtime.GOARCH == "ppc64le" {
+		_, err := o.ExecPrivilegeCommand(o.logWriter, "bootlist", "-m", "normal", "-o", device)
+		if err != nil {
+			o.log.WithError(err).Errorf("Failed to set boot disk with bootlist. Skipping...")
+		}
+		return nil
```

Comment:
Do all ppc64le architecture servers have the bootlist command, or just particular IBM machines? If it's the latter, maybe we should only `return nil` within an `else` statement

Reply:
All RHCOS for ppc64le has the bootlist command.